tests/profiler/test_profiler.py

```diff
@@ -431,6 +433,29 @@ def test_pytorch_profiler_nested(tmpdir):
     assert events_name == expected, (events_name, torch.__version__, platform.system())
 
 
+def test_pytorch_profiler_loggercollection(tmpdir):
```

Comment:
Hey! Thanks for the PR. 
This test seems useful, however the fix itself is quite independent of the profiling. I would like to suggest adding a small test to `tests/trainer/properties/log_dir/` for the logger collection save path.

Reply:
Done!